content/en/users/check-in/vos/_index.md

```diff
@@ -382,6 +382,170 @@ be supported in the future.
 
   Same as the update but requires setting the membership status to `Deleted`
 
+## VO membership API v2
+
+Check-in provides a REST API that allows clients to manage membership
+information only for the VOs they are authoritative for.
+
+Features:
```

Comment:
Add note about the ability to manage role titles

Reply:
Done